### DISCUSSION TITLE
Is anyone using Keystatic in production already?

### DISCUSSION THREAD

**Discussion by simonswiss** [Thinkmill/keystatic#159] [Show and tell]
Is anyone brave enough to be driving some production project with Keystatic?

It's admittedly not recommended just yet, but... I'll start: 

https://betterdevscreencasts.com is powered by Keystatic!

I use this low-risk side project to learn (and teach) Keystatic. I regularly upgrade it to the latest version of Keystatic, which leads to a bit of refactoring, but also great opportunities to find "teachable nuggets" to help the community use Keystatic.

![CleanShot 2023-03-13 at 10 39 00@2x](https://user-images.githubusercontent.com/485747/224581073-f7869c32-b0fd-46b6-9fba-3eac5a4045cc.png)

**Comment by UsamaInsights** [comment]
Hey  @simonswiss ,
Can you provide the code of your site if you don't mind, I am also starting my own site with Keystatic so your site code will save me some time.
I will also link your site as a contributer.
Warm Regards,

**Reply by simonswiss** [reply]
Hey!

Sorry it took me so long to get back. I have made the [Better Dev Screencasts repo](https://github.com/simonswiss/better-dev) public. 

I haven't touched the code in a while and it's maybe not the best example of Keystatic at work, but hopefully this shoulxd help — and please ask if you have questions!

**Reply by simonswiss** [reply]
@UsamaInsight in case this is still useful 👆

**Comment by etmartinkazoo** [comment]
My business [Brem](https://brem.io) has a site in production using it. Client loves it for content management: [The Desenberg](https://desenbergevents.com)

Keystatic on the backend and Astro on the frontend. It is a lovely combination. We have a number of projects in progress at the moment for marketing sites that use hybrid rendering with Astro and we are going to use Keystatic for content management on all of them. Bravo @simonswiss

**Reply by simonswiss** [reply]
Whoaaa this is amazing to hear! 🎉

**Reply by MrMacStripe** [reply]
I have built the first production ready project for a customer as well and I am currently figuring out the way to go for deployment. 

How did you do the hand over to the client? My build process takes about 90 seconds right now for each build. If you change multiple sites on one go, you trigger 5 rebuilds for example, so the final result takes time to be available. I am worried that this is hard to bring across as there is no indication (for the client) on when the latest built has finished.

**Reply by solrik** [reply]
@etmartinkazoo 
This sounds really cool.
Does your client have knowledge on PR and deployment? 
And I tried to access the [link](https://desenbergevents.com/keystatic) you mentioned, it shows 404. How did you block this route?

**Reply by etmartinkazoo** [reply]
@solrik - Unfortunately the repo was private so we needed to add paid seats. Because of this we pivoted to DecapCMS for this project. Perhaps I am missing something obvious but paying for user seats on our private repos wasn't a good fit for this particular project.

**Reply by dralshehri** [reply]
Maybe adding support for [git-gateway](https://github.com/netlify/git-gateway) would solve such an issue.

**Comment by zakiego** [comment]
Is a personal website also included? 😅

If yes, my personal website https://zakiego.com/articles/v4 uses Keystatic as the content management system. I also tweeted about Keystatic here https://twitter.com/zakiego/status/1678041579142139904

**Reply by simonswiss** [reply]
Yes, it absolutely counts as well 🤣

Very nice site — great use of Tailwind UI's Next.js template ✨

Looks like you're doing stuff with Zod and Keystatic, that's really interesting!

@JedWatson and the team were actually talking about doing some Astro content collection Zod-to-Keystatic-config stuff this morning 😅

**Comment by samithaf** [comment]
@simonswiss we use Keystatic in Westpac design system docs https://gel.westpacgroup.com.au/

**Reply by amxv** [reply]
now thats a well-designed website :)

**Comment by Matthijz98** [comment]
I use it for: 
https://kleinvaarbewijstheorie.nl/ - for all content
https://sliceofbits.com/ - For now only the projects and tech sections
For both i hosted keystatic on a other url because the production sites are hosted on nginx

**Comment by sorhavn** [comment]
I just finished creating a simple blog platform for my friends. Only demo is live for now, but the code is there, so if anyone's interested, give it a try, look at the code and let me know whatever you have in mind. Thanks ❤️ 

Code: https://github.com/blueambr/writeup
Demo: https://bwriteup.vercel.app/
